### PR TITLE
Inventory - Update to v4.2.1

### PR DESCRIPTION
**Pull request by texasRanger09**
fixes part of https://github.com/getditto/ditto/issues/8942

### CODE REVIEW

File: .env.template [hunk 1]
```diff
@@ -1,2 +1,2 @@
 APP_ID=
-OFFLINE_LICENSE_TOKEN=
+ONLINE_AUTH_TOKEN=
```
Comment (okdistribute) [inline .env.template:2]: Is this an accident?

**Review by okdistribute — CHANGES_REQUESTED** [review]

File: .env.template (hunk 1 above)
Comment (texasRanger09) [reply .env.template:2]: The code requests the Online auth token and not the offline license token, so I changed the .env file to match and to stay inline with what we have for Android inventory

Comment (okdistribute) [reply .env.template:2]: Thanks!

**Review by kndoshn — APPROVED** [review]

**Comment by kndoshn** [comment]
@okdistribute  I've submitted this to the review - please approve and merge.

**Pull request merged**